target/linux/ramips/files/drivers/net/ethernet/ralink/gsw_mt7620.c

```diff
@@ -285,11 +287,17 @@ static int mt7620_gsw_probe(struct platform_device *pdev)
 	return 0;
 }
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6,11,0)
```

Comment:
this _remove does nothing. Should be removed.

Reply:
calls platform_set_drvdata(pdev, NULL);

Reply:
Which happens anyway.

Reply:
I think all files in the ralink folder need to be reviewed. I found for example that in mt7530.c there is an unnecessary include list.h
As I have already written, I propose a separate PR.